server/app/uk/gov/ons/addressIndex/server/controllers/AddressIndexController.scala

```diff
@@ -1,5 +1,7 @@
 package uk.gov.ons.addressIndex.server.controllers
 
-import play.api.mvc.Controller
+import uk.gov.ons.addressIndex.server.modules.AddressIndexCannedResponse
 
-abstract class AddressIndexController extends Controller
\ No newline at end of file
+abstract class AddressIndexController
```

Comment:
This is an abstract controller layer; idea is to add to this as opposed to having a load of common extends and mixins.